src/app/core/dashboard/dashboard-list-widget/dashboard-list-widget.component.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  AfterViewInit,
+  Component,
+  ContentChild,
+  Input,
+  OnChanges,
+  SimpleChanges,
+  ViewChild,
+} from "@angular/core";
+import { IconName } from "@fortawesome/fontawesome-svg-core";
+import { DashboardTheme } from "../dashboard-widget/dashboard-widget.component";
+import { MatTable, MatTableDataSource } from "@angular/material/table";
+import { MatPaginator } from "@angular/material/paginator";
+import { BehaviorSubject } from "rxjs";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { filter, map } from "rxjs/operators";
+import { applyUpdate } from "../../entity/model/entity-update";
+
+/**
+ * Base dashboard widget to build widgets that display a number of entries as a table.
+ *
+ * This widget automatically handles loading animation and pagination.
+ * Define a mat-table with your desired layout for displaying the entries through content-projection
+ * and provide the data array as an input using async pipe.
+ *
+ * <app-dashboard-list-widget [entries]="notes | async">
+ *   <table mat-table>
+ *     <!-- define row layout here-->
+ *   </table>
+ * </app-dashboard-list-widget>
+ *
+ * To highlight a row with a colored bar at the front, apply `class="row-indicator"` to your first <td>
+ *   and set the css variable to the desired color, e.g. `[ngStyle]="{'--row-indicator-color': row.getColor?.()}"`
+ */
+@Component({
+  selector: "app-dashboard-list-widget",
+  templateUrl: "./dashboard-list-widget.component.html",
+  styleUrls: ["./dashboard-list-widget.component.scss"],
+})
+export class DashboardListWidgetComponent<E>
+  implements OnChanges, AfterViewInit
+{
+  @Input() subtitle: string;
+  @Input() icon: IconName;
```

Comment:
```suggestion
  @Input() icon: IconName = "exclamation-triangle";
```
This way no `undefined` value would be passed to the `FaDynamicIconComponent`